Compile error on 1.13.0-dev in fastled_base

<!-- Thanks for reporting a bug for this project. READ THIS FIRST:

- Provide as many details as possible. Simply saying "X gives bug" or "X gives error" is not enough!
- Paste logs, configuration sample and code into the backticks (```).
- Read through the template carefully and fill out all missing details.
- Please also search for similar issues in this issue tracker first and read through the ESPHome FAQ.

DO NOT DELETE ANY TEXT from this template! Otherwise the issue may be closed without a comment.
-->

**Operating environment/Installation (Hass.io/Docker/pip/etc.):**
<!--
Please provide details about your environment below this line. -->

Hassio 0.92 , running on 32bit debian 9 in docker

**ESP (ESP32/ESP8266, Board/Sonoff):**
<!--
Please provide details about which ESP you're using below.
-->
ESP32 NodeMCU board from china

**Affected component:**
<!--
Please add the link to the documentation at https://esphome.io/index.html of the component in question.
-->
https://esphome.io/components/light/fastled.html

**Description of problem:**
i'm trying to compile fastleds on 1.13.0-dev (i have upgraded, because i was having timeout troubles with OTA updates on 1.12). It compiled ok before.


**Problem-relevant YAML-configuration entries:**
```yaml
esphome:
  name: esp32_001
  platform: ESP32
  board: nodemcu-32s

wifi:
  ssid: "devices"
  password: "xxx"
  use_address: "192.168.x.y"

# Enable logging
logger:
  level: DEBUG
  
# Enable Home Assistant API
api:
  password: 'xzzx'

ota:
  password: 'zxx'

#power_supply:
#  - id: 'atx_power_supply'
#    pin:
#      number: GPIO27
#      inverted: true

switch:
  - platform: gpio
    name: "Living Room Dehumidifier"
    pin: GPIO22

light:
  - platform: fastled_clockless
    chipset: WS2811
    pin: GPIO21
    num_leds: 60
    rgb_order: BGR
    name: "FastLED WS2811 Light"

```

**Traceback (if applicable):**
<!--
Please copy the traceback here if compilation is failing. If possible, also connect to the ESP over USB and copy its logs into the backticks.
-->
```
src/main.cpp:50:1: error: 'fastled_base' does not name a type
fastled_base::FastLEDLightOutput *fastled_base_fastledlightoutput;
^
src/main.cpp: In function 'void setup()':
src/main.cpp:83:3: error: 'fastled_base_fastledlightoutput' was not declared in this scope
fastled_base_fastledlightoutput = new fastled_base::FastLEDLightOutput();
^
src/main.cpp:83:41: error: 'fastled_base' does not name a type
fastled_base_fastledlightoutput = new fastled_base::FastLEDLightOutput();
^
src/main.cpp:95:45: error: 'WS2811' was not declared in this scope
fastled_base_fastledlightoutput->add_leds<WS2811, 21, BGR>(60);
^
src/main.cpp:95:57: error: 'BGR' was not declared in this scope
fastled_base_fastledlightoutput->add_leds<WS2811, 21, BGR>(60);
^
In file included from src/esphome/components/api/service_call_message.h:4:0,
from src/esphome/components/api/api_server.h:14,
from src/main.cpp:4:
src/esphome/core/automation.h: In instantiation of 'bool esphome::Trigger<Ts>::is_running() [with Ts = {}]':
src/esphome/components/light/base_light_effects.h:72:34:   required from here
src/esphome/core/automation.h:58:48: error: request for member 'is_running' in '((esphome::Trigger<>*)this)->esphome::Trigger<>::automation_parent_', which is of pointer type 'esphome::Automation<>*' (maybe you meant to use '->' ?)
return this->automation_parent_.is_running();
^
```

**Additional information and things you've tried:**




<!-- LEAVE THIS LINE AS-IS AND DON'T DELETE IT, OTHERWISE THE ISSUE WILL BE CLOSED AUTOMATICALLY. -->

--- a/esphome/core/automation.h
+++ b/esphome/core/automation.h
@@ -55,7 +55,7 @@ template<typename... Ts> class Trigger {
   bool is_running() {
     if (this->automation_parent_ == nullptr)
       return false;
-    return this->automation_parent_.is_running();
+    return this->automation_parent_->is_running();
   }
 
  protected:

